tests/unit/mainwindow/statusbar/test_url.py

```diff
@@ -0,0 +1,126 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2014-2016 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
```

Comment:
nitpick, that should just be 2016 (and maybe your name)... I really need to find a way to autogenerate those, as distributions usually require them